server/routes/referrals/referralFormPresenter.test.ts

```diff
@@ -108,7 +108,7 @@ describe('ReferralFormPresenter', () => {
           usingRarDays: true,
           maximumRarDays: 10,
         })
-        const presenter = new ReferralFormPresenter(referral)
+        const presenter = new ReferralFormPresenter(referral, 'Accommodation')
```

Comment:
The rest of the application assumes that a `ServiceCategory`'s `name` is always in lower case - i.e. that it's suitable to drop into a sentence. I think we could do with verifying this in a contract test, actually.

Reply:
Any suggestions for how to write a better contract test than the one we have? the current one here https://github.com/ministryofjustice/hmpps-interventions-ui/blob/e394d6a6bf2fea27c31c38bfe931cde11500db9d/server/services/interventionsService.test.ts#L862 kind of tests it, but not all that explicitly.

Reply:
We could use a [regular expression matcher](https://docs.pact.io/getting_started/matching/#regular-expressions) on the `name` field in that test, to check that it starts with a capital letter (`/^[A-Z].*/`)?

Reply:
Didn't know that was a thing, have added it - thanks!